week3_model_free/sarsa.ipynb

```diff
@@ -328,7 +328,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Let's now see what did the algorithms learn by visualizing their actions at every state."
+    "Let's now see, what did the algorithms learn by visualizing their actions at every state."
```

Comment:
Redundant comma?

It should also be "what the algorithms learned" instead of "what did the algorithms learn".